app/models/tenant.rb

```diff
@@ -17,6 +17,6 @@ class Tenant < ApplicationRecord
   has_many :tasks
 
   def self.tenancy_enabled?
-    ENV["BYPASS_TENANCY"].blank?
+    ClowderConfig.instance["BYPASS_TENANCY"].blank?
```

Comment:
@hsong-rh Can this just return True, the BYPASS_TENANCY was when we were developing it initially. Now everything is Tenant based.

Reply:
removed